tests/event_store/test_subscription.py

```diff
@@ -419,6 +420,68 @@ def test_receiving_in_batches(
         then(subscription).next_batch_is_empty()
 
 
+@pytest.mark.skip_backend(
+    backend=["esdb", "in_memory"],
+    reason="Required only for SQL-based backends with transactions",
+)
+class TestHandlingUncommittedMessagesFromOtherClients:
```

Comment:
There are no tests for other types of subscription (category and events)

Reply:
(clarified on Discord) - this is about lacking tests using filters for category and event type